tests/tcp_listener.rs

```diff
@@ -343,6 +343,34 @@ fn try_clone_different_poll() {
     thread_handle2.join().expect("unable to join thread");
 }
 
+#[test]
+fn platform_specific_data_print() {
```

Comment:
I don't really think there is a need for a test as the output is not stable.

If you do want a test I would suggest making one per platform the there `event` module and manually creating the `Event`s.

Reply:
Removed the test, not worth the trouble atm.